tensorflow/tools/dockerfiles/partials/mpi.partial.Dockerfile

```diff
@@ -0,0 +1,44 @@
+# install libnuma, openssh, wget
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
+        libnuma-dev \
+        openssh-server \
+        openssh-client \
+        wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* || \
+    yum -y update && yum -y install \
+            numactl-devel \
+            openssh-server \
+            openssh-clients \
+            wget && \
+    yum clean all || \
+    echo "Unsupported Linux distribution. Aborting!" && exit 1
+
+# Install Open MPI
+RUN mkdir /tmp/openmpi && \
+    cd /tmp/openmpi && \
+    wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-4.0.0.tar.gz && \
```

Comment:
Why horovod is to pull the newest master branch from git repo, but the openmpi is to get a specified revision? Couldn't it get from https://github.com/open-mpi/ompi.git master too?